index.html

```diff
@@ -0,0 +1,240 @@
+                </div>
+                <div class="team-member">
+                    <img src="./assets/images/sarahLee.jpg" alt="Sarah Lee">
+                    <h3>Sarah Lee</h3>
+                    <p><strong>Marketing Director</strong></p>
+                    <p>Sarah is responsible for creating and executing marketing strategies to increase brand awareness. Her expertise in digital marketing has helped us reach new heights.</p>
+                    <ul>
+                        <li><a href="https://linkedin.com" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+                        <li><a href="https://twitter.com" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                    </ul>
+                </div>
+                <div class="team-member">
+                    <img src="./assets/images/michaelBrown.jpg" alt="Michael Brown">
+                    <h3>Michael Brown</h3>
+                    <p><strong>Designer</strong></p>
+                    <p>Michael brings a unique creative vision to our team. He’s responsible for making sure our products not only function well but look great too!</p>
+                    <ul>
+                        <li><a href="https://dribbble.com" target="_blank"><i class="fab fa-dribbble"></i></a></li>
+                        <li><a href="https://twitter.com" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                    </ul>
+                </div>
+                <div class="team-member">
+                    <img src="./assets/images/emmaWilson.jpg" alt="Emma Wilson">
+                    <h3>Emma Wilson</h3>
+                    <p><strong>Project Manager</strong></p>
+                    <p>Emma ensures all our projects stay on track and meet deadlines. With her sharp organizational skills, she keeps us efficient and focused on delivering high-quality results.</p>
+                    <ul>
+                        <li><a href="https://linkedin.com" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+                        <li><a href="https://twitter.com" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        
+        <section class="section" id="contact">
+            <h2>Contact Us At</h2>
+          <div>
+            <form action="post" id="contact-form">
+                <label for="name">Name</label>
+                <input type="text" name="name" id="name" />
+                <label for="email">Email</label>
+                <input type="email" name="email" id="email" />
+                <label for="message">Message</label>
+                <textarea
+                  name="message"
+                  id="message"
+                  cols="30"
+                  rows="10"
+                ></textarea>
+                <button>Send</button>
+              </form>
+              <img src="./assets/images/duck.gif" alt="" id="walkingDuck"/>
```

Comment:
Optimize images for web use to improve performance. Use alt attributes for accessibility.